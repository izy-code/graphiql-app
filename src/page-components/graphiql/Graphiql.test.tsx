import { render } from '@testing-library/react';

import GraphiQlPage from '@/app/[locale]/graphiql/[[...graphiql]]/page';

vi.mock('next/navigation', async (importOriginal) => {
  const actual = await importOriginal<object>();

  return {
    ...actual,
    useRouter: vi.fn(() => ({ push: vi.fn((arg: string): string => arg) })),
  };
});

vi.mock('@/hooks/useAuth', () => ({ useAuth: vi.fn((): boolean => true) }));

describe.skip('GraphQL page', () => {
  it('renders correctly', () => {
    const { container } = render(<GraphiQlPage />);

    expect(container).toMatchSnapshot();
  });
});
