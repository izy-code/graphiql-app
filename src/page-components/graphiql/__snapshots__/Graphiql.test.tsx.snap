// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GraphQL page > renders correctly 1`] = `
<div>
  <div
    class="_page_d98ae5"
  >
    <div
      class="_container_d98ae5"
    >
      <h1
        class="_title_d98ae5"
      >
        GraphiQl Client
      </h1>
      <div
        class="_items_d98ae5"
      >
        <h2
          class="_sectionTitle_d98ae5"
        >
          URL
        </h2>
        <form
          autocomplete="off"
          class="_inputContainer_71d148 MuiBox-root css-vyiutb"
          novalidate=""
        >
          <div
            class="MuiFormControl-root MuiTextField-root _input_71d148 css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-standard css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="false"
              for=":r0:"
              id=":r0:-label"
            >
              Endpoint URL
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl css-1eed5fa-MuiInputBase-root-MuiInput-root"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                id=":r0:"
                type="text"
                value=""
              />
            </div>
          </div>
        </form>
        <form
          autocomplete="off"
          class="_inputContainer_71d148 MuiBox-root css-vyiutb"
          novalidate=""
        >
          <div
            class="MuiFormControl-root MuiTextField-root _input_71d148 css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-standard css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="false"
              for=":r1:"
              id=":r1:-label"
            >
              SDL URL
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl css-1eed5fa-MuiInputBase-root-MuiInput-root"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                id=":r1:"
                type="text"
                value=""
              />
            </div>
          </div>
        </form>
      </div>
      <div
        class="_section_d98ae5"
      >
        <h2
          class="_sectionTitle_d98ae5"
        >
          Params
        </h2>
        <div
          class="_container_3640f1"
        >
          <div
            class="_headerContainer_3640f1"
          >
            <h4
              class="_subtitle_3640f1"
            >
              Headers:
            </h4>
            <div
              class="_inputContainer_3640f1"
            >
              <form
                autocomplete="off"
                class="_inputContainer_71d148 MuiBox-root css-1l5xa58"
                novalidate=""
              >
                <div
                  class="MuiFormControl-root MuiTextField-root _input_71d148 css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
                >
                  <label
                    class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for=":r2:"
                    id=":r2:-label"
                  >
                    Key
                  </label>
                  <div
                    class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                      id=":r2:"
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        class="css-yjsfm1"
                      >
                        <span>
                          Key
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </form>
              <form
                autocomplete="off"
                class="_inputContainer_71d148 MuiBox-root css-1l5xa58"
                novalidate=""
              >
                <div
                  class="MuiFormControl-root MuiTextField-root _input_71d148 css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
                >
                  <label
                    class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for=":r3:"
                    id=":r3:-label"
                  >
                    Value
                  </label>
                  <div
                    class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                      id=":r3:"
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        class="css-yjsfm1"
                      >
                        <span>
                          Value
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </form>
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary _button_3640f1 css-sghohy-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                + row
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root _table_3640f1 css-1cjjkw7-MuiPaper-root-MuiTableContainer-root"
          >
            <table
              aria-label="headers table"
              class="MuiTable-root css-d7jkuz-MuiTable-root"
            >
              <thead
                class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
              >
                <tr
                  class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                    scope="col"
                  >
                    Key
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1azl6jz-MuiTableCell-root"
                    scope="col"
                  >
                    Value
                  </th>
                </tr>
              </thead>
              <tbody
                class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
              >
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                    scope="row"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":r4:"
                          placeholder="Header Key"
                          type="text"
                          value="Cache-control"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-177gid-MuiTableCell-root"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":r5:"
                          placeholder="Header Value"
                          type="text"
                          value="no-cache"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                    scope="row"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":r6:"
                          placeholder="Header Key"
                          type="text"
                          value="Host"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-177gid-MuiTableCell-root"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":r7:"
                          placeholder="Header Value"
                          type="text"
                          value="calculated when request is sent"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                    scope="row"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":r8:"
                          placeholder="Header Key"
                          type="text"
                          value="User-Agent"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-177gid-MuiTableCell-root"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":r9:"
                          placeholder="Header Value"
                          type="text"
                          value="PostmanRuntime/7.41.1"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                    scope="row"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":ra:"
                          placeholder="Header Key"
                          type="text"
                          value="Accept"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-177gid-MuiTableCell-root"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":rb:"
                          placeholder="Header Value"
                          type="text"
                          value="*/*"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                    scope="row"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":rc:"
                          placeholder="Header Key"
                          type="text"
                          value="Accept-Encoding"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-177gid-MuiTableCell-root"
                  >
                    <div
                      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_3640f1 css-wb57ya-MuiFormControl-root-MuiTextField-root"
                    >
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                      >
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                          id=":rd:"
                          placeholder="Header Value"
                          type="text"
                          value="gzip, deflate, br"
                        />
                        <fieldset
                          aria-hidden="true"
                          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                        >
                          <legend
                            class="css-ihdtdm"
                          >
                            <span
                              class="notranslate"
                            >
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
        <div
          class="_item_d98ae5"
        >
          <h4>
            Query: 
          </h4>
          <form
            autocomplete="off"
            class="_inputContainer_a5b21b MuiBox-root css-nios8u"
            novalidate=""
          >
            <div
              class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_a5b21b css-1exn6sj-MuiFormControl-root-MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
                data-shrink="false"
                for=":re:"
                id=":re:-label"
              >
                Query
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline css-8ewcdo-MuiInputBase-root-MuiOutlinedInput-root"
              >
                <textarea
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline css-1sqnrkk-MuiInputBase-input-MuiOutlinedInput-input"
                  id=":re:"
                  rows="4"
                  style="height: 0px; overflow: hidden;"
                />
                <textarea
                  aria-hidden="true"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline css-1sqnrkk-MuiInputBase-input-MuiOutlinedInput-input"
                  readonly=""
                  style="visibility: hidden; position: absolute; overflow: hidden; height: 0px; top: 0px; left: 0px; transform: translateZ(0); padding-top: 0px; padding-bottom: 0px; width: 100%;"
                  tabindex="-1"
                />
                <fieldset
                  aria-hidden="true"
                  class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="css-yjsfm1"
                  >
                    <span>
                      Query
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </form>
        </div>
        <div
          class="_item_d98ae5"
        >
          <h4>
            Variables: 
          </h4>
          <form
            autocomplete="off"
            class="_inputContainer_a5b21b MuiBox-root css-nios8u"
            novalidate=""
          >
            <div
              class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root _input_a5b21b css-1exn6sj-MuiFormControl-root-MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
                data-shrink="false"
                for=":rf:"
                id=":rf:-label"
              >
                Variables
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline css-8ewcdo-MuiInputBase-root-MuiOutlinedInput-root"
              >
                <textarea
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline css-1sqnrkk-MuiInputBase-input-MuiOutlinedInput-input"
                  id=":rf:"
                  rows="4"
                  style="height: 0px; overflow: hidden;"
                />
                <textarea
                  aria-hidden="true"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline css-1sqnrkk-MuiInputBase-input-MuiOutlinedInput-input"
                  readonly=""
                  style="visibility: hidden; position: absolute; overflow: hidden; height: 0px; top: 0px; left: 0px; transform: translateZ(0); padding-top: 0px; padding-bottom: 0px; width: 100%;"
                  tabindex="-1"
                />
                <fieldset
                  aria-hidden="true"
                  class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="css-yjsfm1"
                  >
                    <span>
                      Variables
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </form>
        </div>
      </div>
    </div>
    <div
      class="_section_d98ae5"
    >
      <h2
        class="_sectionTitle_d98ae5"
      >
        Response
      </h2>
      <div
        class="_oneLine_d98ae5"
      >
        <h4>
          Status:
        </h4>
        200
      </div>
      <div
        class="_body_d98ae5"
      >
        <h4>
          Body:
        </h4>
        <pre
          class="MuiBox-root css-eky454"
        >
          {}
        </pre>
      </div>
    </div>
  </div>
</div>
`;
